### DISCUSSION TITLE
能否增加一个二维码识别功能

### DISCUSSION THREAD

**Discussion by banzhiz** [shuax/MouseInc.Settings#232] [Ideas]
如果能像截图ocr一样框选区域识别二维码就好了，毕竟电脑上不如手机识别二维码那么方便，现在电脑上截图保存在上传识别二维码的网站流程实在是太麻烦了，用手机识别的话还要复制到电脑上再打开。
希望作者考虑一下。

**Comment by shuax** [comment]
很早之前是有这个功能的，识别效果不好，已经砍掉了。

**Reply by tangjyan** [reply]
微信PC版已经有了这个功能了

**Comment by qqqqcat1222** [comment]
我也觉得很有必要搞个二维码识别

**Comment by qqqqcat1222** [comment]
微信官方开源了自家优化的二维码引擎！3行代码让你拥有微信扫码能力，不知道这个是否有用。https://mp.weixin.qq.com/s/xzc19NIGxy_YTLyw2_DCoA

**Reply by gbsix** [reply]
这个功能会很实用

**Reply by ikarosu** [reply]
@shuax 大佬看看这个，能不能用这个东西加回来这个功能

**Comment by gbsix** [comment]
这个功能都有哪些使用场景？ 
我觉得生成二维码功能是必须，方便手机扫描。二维码一般都要求打开手机app扫描，电脑上就是能识别二维码但不是手机客户端大多也不允许你打开/跳转吧
